slices/fonts-dejavu-mono.yaml

```diff
@@ -12,8 +12,6 @@ slices:
       /usr/share/fonts/truetype/dejavu/DejaVuSansMono.ttf:
 
   config:
-    essential:
-      - fonts-dejavu-mono_fonts
```

Comment:
same here. Whoever uses those already would have a regression.

If you want a slice with just configs, then the best is to simply create a new one. Something like `configs-only`.

Reply:
Okay, I will redesign a bit to keep compatibility